Problem when running the project

I clone the app on my computer, and after installing the dependencies by using 'npm install' and running the project by using 'npm run dev-server', i end up with the logs above :
> html-dom@ dev-server C:\laragon\www\github-projects\html-dom
> npm run copy && webpack-dev-server


> html-dom@ copy C:\laragon\www\github-projects\html-dom
> npm run copy:public && npm run copy:demo


> html-dom@ copy:public C:\laragon\www\github-projects\html-dom
> rimraf dist && mkdir dist && cp -rf public/* dist


> html-dom@ copy:demo C:\laragon\www\github-projects\html-dom
> cp -rf demo dist && NODE_ENV=production postcss vendors/tailwind@1.1.4/tailwind.css -o dist/demo/styles.css

'NODE_ENV' n’est pas reconnu en tant que commande interne
ou externe, un programme exécutable ou un fichier de commandes.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! html-dom@ copy:demo: `cp -rf demo dist && NODE_ENV=production postcss vendors/tailwind@1.1.4/tailwind.css -o dist/demo/styles.css`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the html-dom@ copy:demo script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\KDragon\AppData\Roaming\npm-cache\_logs\2020-04-02T13_20_53_205Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! html-dom@ copy: `npm run copy:public && npm run copy:demo`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the html-dom@ copy script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\KDragon\AppData\Roaming\npm-cache\_logs\2020-04-02T13_20_53_307Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! html-dom@ dev-server: `npm run copy && webpack-dev-server`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the html-dom@ dev-server script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\KDragon\AppData\Roaming\npm-cache\_logs\2020-04-02T13_20_53_435Z-debug.log

Is there any problem here because its said that the problem not in my npm ?

--- a/package.json
+++ b/package.json
@@ -24,7 +24,7 @@
     "scripts": {
         "copy": "npm run copy:public && npm run copy:demo",
         "copy:public": "rimraf dist && mkdir dist && cp -rf public/* dist",
-        "copy:demo": "cp -rf demo dist && NODE_ENV=production postcss vendors/tailwind@1.1.4/tailwind.css -o dist/demo/styles.css",
+        "copy:demo": "cp -rf demo dist && SET NODE_ENV=production postcss vendors/tailwind@1.1.4/tailwind.css -o dist/demo/styles.css",
         "dev": "npm run copy && webpack --mode development",
         "dev-server": "npm run copy && webpack-dev-server",
         "build": "npm run copy && webpack --mode production && npm run export",

